project/frontend/src/components/ChartCard/ChartCard.module.css

```diff
@@ -0,0 +1,15 @@
+.ChartContainer {
+    list-style: none;
+    height: 90%;
+    min-width: 60%;
+    border: 1px solid #9a9a9a;
+    margin: 0px 25px;
+    background-color: rgba(255, 255, 255, 0.45);
+    border-radius: 7px;
+    transition: transform 0.4s ease;
```

Comment:
what's the transition for?

Reply:
:( I copy-pasted your code from another component

Reply:
Looks like it might be the comp card to me.. the comp cards zoom in if you hover over them on the compselect page, hence the transition, dun need this for chart, same for the margin and list style

Reply:
Oh or is that the header bar innernav items... either way yeah

Reply:
fixed